lib/openstack_taster.rb

```diff
@@ -10,7 +10,7 @@
 # @author Andrew Tolvstad, Samarendra Hedaoo, Cody Holliday
 class OpenStackTaster
   INSTANCE_FLAVOR_NAME = 'm1.tiny'
-  INSTANCE_NETWORK_NAME = 'public'
+  INSTANCE_NETWORK_NAME = 'public1'
```

Comment:
Do we only intend to use `openstack_taster` on https://oprod-controller1.osuosl.org/ (or OpenStack instances where we have a network named `public1`)?

Perhaps this would be better off as a configurable value? (Not necessarily recommending that should be a part of this PR.)